src/components/GithubLeaderboard.tsx

```diff
@@ -56,13 +59,42 @@ export const GithubLeaderboard: React.FC<Props> = ({
   ];
 
   const headers = [
-    { key: 'rank', label: 'Rank' },
-    { key: 'username', label: 'Username' },
+    { key: 'rank', label: 'Rank', static: true },
+    { key: 'username', label: 'Username', static: true },
     { key: 'totalCommits', label: 'Commits' },
     { key: 'totalPrs', label: 'PRs' },
     { key: 'followers', label: 'Followers' },
   ];
 
+  const getOrderByFromKey = (key: string): GitHubOrderBy | null => {
```

Comment:
do we have a type that this string adheres to as a key of?

Reply:
I don't think so?

Reply:
a lot of the `string`s can be replaced, something like this maybe:

https://github.com/swecc-uw/swecc-leaderboard/commit/d85542ca33fa1917a769b7e953babe02085418b2

although it isn't strictly needed. Can cherry-pick/merge this commit if you want, otherwise lgtm